pkg/pxc/app/statefulset/node.go

```diff
@@ -195,6 +195,8 @@ func (c *Node) AppContainer(spec *api.PodSpec, secrets string, cr *api.PerconaXt
 				Name:          "mysql-admin",
 			},
 		)
+		appc.ReadinessProbe.Exec.Command = []string{"/var/lib/mysql/readiness-check.sh"}
```

Comment:
it should be only for 1.6.0

Reply:
@delgod  see line https://github.com/percona/percona-xtradb-cluster-operator/pull/557/files#diff-555e0ded4b10474fc1bb6e81e71befffL190 , now it is for 1.6.0 only